examples/randomness/src/main.py

```diff
@@ -0,0 +1,8 @@
+import _random
```

Comment:
I don't understand why there's an underscore. I don't use python very often so maybe I'm just missing something, but every piece of documentation I'm googling shows you use `random` not `_random`. Also, I didn't know you had to create an object and then call random on it. I was expecting:

```python
from random import random

@update
def random_number() -> float64:
    return random()
```

Why is it not like that?

Reply:
We don't have the actual random module from the stdlib yet, we just have the native (Rust in our case) implementation module, just enough to test the randomness